python/ledger_tutorial.py

```diff
@@ -41,4 +41,15 @@
         if target_table in public_tables:
             for key, value in public_tables[target_table].items():
                 target_table_changes += 1  # A key was changed
+
+        # extracting transactions on a sample table
+        # Ledger verification is happening implicitly in ccf.ledger.Ledger()
+        if "public:sample.logs" in public_tables:
+            public_tpal_table = public_tables["public:sample.logs"]
```

Comment:
This references a table that doesn't exist in the framework. In this case, using the `target_table` (defined on line `30`) would be simpler/more generic.